mgmt_cli_test.py

```diff
@@ -179,16 +179,18 @@ def verify_backup_success(self, mgr_cluster, backup_task, keyspace_name='keyspac
         self.restore_backup_from_backup_task(mgr_cluster=mgr_cluster, backup_task=backup_task,
                                              keyspace_and_table_list=per_keyspace_tables_dict)
 
-    def _generate_load(self):
+    def _generate_load(self, keyspace_name_to_replace=None):
         self.log.info('Starting c-s write workload')
         stress_cmd = self.params.get('stress_cmd')
+        if keyspace_name_to_replace:
+            stress_cmd = stress_cmd.replace("keyspace1", keyspace_name_to_replace)
         stress_thread = self.run_stress_thread(stress_cmd=stress_cmd)
         self.log.info('Sleeping for 15s to let cassandra-stress run...')
         time.sleep(15)
         return stress_thread
 
-    def generate_load_and_wait_for_results(self):
-        load_thread = self._generate_load()
+    def generate_load_and_wait_for_results(self, keyspace_name_to_replace=None):
+        load_thread = self._generate_load(keyspace_name_to_replace=keyspace_name_to_replace)
         load_results = load_thread.get_results()
```

Comment:
will be the test's timeout enough to add another load in parallel like you are doing here?

Reply:
YEah, the test passed in all distributions. It's not a very long stress.

Reply:
does it run in parallel with other c-s commands?
where can i see the test's results?

Reply:
Not in parallel, only when I call the function
TEST_ID=4bdc9279-5e25-4103-b99e-c0f106deaad6